tests/setup_nodes.py

```diff
@@ -33,10 +36,16 @@ def cleanup_keyring(keyring: TempKeyring):
     keyring.cleanup()
 
 
-temp_keyring = TempKeyring()
+temp_keyring = TempKeyring(populate=True)
 keychain = temp_keyring.get_keychain()
 atexit.register(cleanup_keyring, temp_keyring)  # Attempt to cleanup the temp keychain
```

Comment:
Shouldn't be necessary anymore, but I understand if you don't want to make that change with this PR